plugins/woocommerce-admin/client/marketing/overview-multichannel/LearnMarketing.tsx

```diff
@@ -0,0 +1,191 @@
+import './LearnMarketing.scss';
+
+type Post = {
+	author_avatar: string;
+	author_name: string;
+	date: string;
+	image: string;
+	link: string;
+	title: string;
+};
+
+type PostTileProps = {
+	post: Post;
+};
+
+const PlaceholderPostTile = () => {
+	return (
+		<div className="woocommerce-marketing-learn-marketing-card__post">
+			<div className="woocommerce-marketing-learn-marketing-card__post-img woocommerce-marketing-learn-marketing-card__post-img--placeholder"></div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-title woocommerce-marketing-learn-marketing-card__post-title--placeholder"></div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-description woocommerce-marketing-learn-marketing-card__post-description--placeholder"></div>
+		</div>
+	);
+};
+
+const PostTile: React.FC< PostTileProps > = ( { post } ) => {
+	return (
+		<a
+			className="woocommerce-marketing-learn-marketing-card__post"
+			href={ post.link }
+			target="_blank"
+			rel="noopener noreferrer"
+			onClick={ () => {
+				recordEvent( 'marketing_knowledge_article', {
+					title: post.title,
+				} );
+			} }
+		>
+			<div className="woocommerce-marketing-learn-marketing-card__post-img">
+				{ post.image && <img src={ post.image } alt="" /> }
+			</div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-title">
+				{ post.title }
+			</div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-description">
+				{
+					// translators: %s: author's name.
+					sprintf( __( 'By %s', 'woocommerce' ), post.author_name )
+				}
+				{ post.author_avatar && (
+					<img
+						src={ post.author_avatar.replace( 's=96', 's=32' ) }
+						alt=""
+					/>
+				) }
+			</div>
+		</a>
+	);
+};
+
+const blogPostCategory = 'marketing';
+const perPage = 2;
+
+const LearnMarketing = () => {
+	const [ page, setPage ] = useState( 1 );
+	const { isLoading, error, posts } = useSelect(
+		( select ) => {
+			const { getBlogPosts, getBlogPostsError, isResolving } =
+				select( STORE_KEY );
+
+			return {
+				posts: getBlogPosts( blogPostCategory ),
+				isLoading: isResolving( 'getBlogPosts', [ blogPostCategory ] ),
+				error: getBlogPostsError( blogPostCategory ),
+			};
+		},
+		[ blogPostCategory ]
+	);
+
+	/**
+	 * Renders card footer.
+	 *
+	 * - If loading is in progress, it returns a loading placeholder in the card footer.
+	 * - If there is an error or there are no posts, there will be no card footer.
+	 * - Returns a pagination component in the card footer for paging through the posts.
+	 */
+	const renderFooter = () => {
+		if ( isLoading ) {
+			return (
+				<div className="woocommerce-marketing-learn-marketing-card__footer--placeholder"></div>
+			);
+		}
+
+		if ( error || posts.length === 0 ) {
+			return null;
+		}
+
+		return (
+			<Pagination
+				showPagePicker={ false }
+				showPerPagePicker={ false }
+				page={ page }
+				perPage={ perPage }
+				total={ posts && posts.length }
```

Comment:
I guess at this point `posts.length` is always defined and greater than 0.

In case post could be undefined. We should do this check before or in line 108 I guess?

Reply:
> We should do this check before or in line 108 I guess?

Yeah, good idea. Done in [`cb7d0e1` (#34264)](https://github.com/woocommerce/woocommerce/pull/34264/commits/cb7d0e125d01be03242b5b3df91e666aa7e0fec4).